flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/JsonRowDeserializationSchema.java

```diff
@@ -340,56 +340,68 @@ private DeserializationRuntimeConverter createFallbackConverter(Class<?> valueTy
 			return Optional.of(createTimeConverter());
 		} else if (simpleTypeInfo == Types.SQL_TIMESTAMP) {
 			return Optional.of(createTimestampConverter());
+		} else if (simpleTypeInfo == Types.LOCAL_DATE) {
+			return Optional.of(this::convertToLocalDate);
+		} else if (simpleTypeInfo == Types.LOCAL_TIME) {
+			return Optional.of(this::convertToLocalTime);
+		} else if (simpleTypeInfo == Types.LOCAL_DATE_TIME) {
+			return Optional.of(this::convertToLocalDateTime);
 		} else {
 			return Optional.empty();
 		}
 	}
 
+	private LocalDate convertToLocalDate(ObjectMapper mapper, JsonNode jsonNode) {
+		return ISO_LOCAL_DATE.parse(jsonNode.asText()).query(TemporalQueries.localDate());
+	}
+
 	private DeserializationRuntimeConverter createDateConverter() {
```

Comment:
Can you modify `createDateConverter` to `convertToDate` style too?

Reply:
Use a unify style?

Reply:
I think this has already been updated. I don't want to change other types converter, because they are clearer now.